csrank/dataset_reader/choicefunctions/choice_data_generator.py

```diff
@@ -15,7 +15,9 @@ def __init__(self, dataset_type="pareto", **kwargs):
             "pareto": self.make_globular_pareto_choices,
         }
         if dataset_type not in dataset_function_options.keys():
```

Comment:
Very minor nitpick at the original code: We sometimes use `obj not in dict.keys()` and sometimes `obj not in dict`.
Both are valid, but I think we should be consistent.
`obj not in dict` is in general regarded as the "more pythonic" syntax.

Reply:
> Both are valid, but I think we should be consistent.

Agreed!

I kind of prefer the more explicit call to `.keys` for readability. It looks like you're right though that the direct `in` is the more "pythonic" way and also more efficient: https://stackoverflow.com/questions/1602934/check-if-a-given-key-already-exists-in-a-dictionary